static-site/src/pages/groups.jsx

```diff
@@ -117,7 +117,10 @@ class GroupsPage extends React.Component {
           resourceType="dataset"
           defaultGroupLinks={true}
           versioned={false}
-          resourceListingCallback={resourceListingCallback}/>
+          resourceListingCallback={resourceListingCallback}
+          // FIXME I do not love this text string; suggestions welcome
+          tooltipText="Click to load the group page"
```

Comment:
[suggestion]

"Click to view this group in isolation, including additional information specific to that group"

Won't look nice if you then tack on `for ${group.groupDisplayName || group.groupName}` but we could change the prop to be a function accepting the group name.

Reply:
The [Nextstrain docs for Groups](https://docs.nextstrain.org/en/latest/learn/groups/index.html#pages) call it a "splash page", so we can follow that pattern here:

"Click to view the individual group splash page"

Reply:
Or something less wordy:

> Go to splash page for blab

> Visit splash page for blab